app/lib/three_scale/semantic_form_builder.rb

```diff
@@ -19,10 +19,12 @@ def inline_errors
       @options_inline_erros || super
     end
 
-    def text_field_with_errors( method, opts = {})
-      errors =  @object.errors[method]
+    def text_field_with_errors(method, opts = {})
+      errors = @object.errors[method]
       if errors.present?
         opts = opts.merge(:class => 'error', 'data-errors' => errors, :title => errors)
+      elsif method == :pattern && @object.pattern.in?(['', '/'])
+        opts = opts.merge(:class => "#{opts[:class]} catch-all-warning")
```

Comment:
You can use arrays with `:class` so you can do

`opts.merge(class: [*opts[:class], 'catch-all-warning'])`

Reply:
`opts = opts.merge` is the same as `opts.merge!` 😄 So considering Hery's idea, the whole line should be:
```ruby
  opts.merge!(class: [*opts[:class], 'catch-all-warning'])
```